<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DominantProcessWidget</class>
 <widget class="QWidget" name="DominantProcessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>152</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Check processes on</string>
       </property>
       <property name="buddy">
        <cstring>cmbCheckProcesses</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbCheckProcesses">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>&lt;local computer&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>&lt;local computer&gt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnCheckProcesses">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../qHostMonitor.qrc">
           <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Exclude process(es)</string>
       </property>
       <property name="buddy">
        <cstring>lnExcludedProcesses</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lnExcludedProcesses"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Alert if proc uses over</string>
       </property>
       <property name="buddy">
        <cstring>spinAlertIf</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QSpinBox" name="spinAlertIf">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>524288</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbAlertIf">
         <item>
          <property name="text">
           <string>CPU (%)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Handles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Threads</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Memory (MB)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VMem (MB)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Address space (MB)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Top process info: show</string>
       </property>
       <property name="buddy">
        <cstring>cmbTopProcess</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmbTopProcess">
       <item>
        <property name="text">
         <string>process ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>process name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>process name, ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>value (Address space)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>value, process ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>value, process name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>value, process name, ID</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3,3,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QCheckBox" name="chkConnectAs">
       <property name="text">
        <string>Connect as:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbConnectLogin">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineConnectPassword"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnConnectAs">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/misc/processMeter.png</normaloff>:/img/misc/processMeter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
